Lib/profiling/sampling/sample.py

```diff
@@ -631,6 +632,9 @@ def sample(
         case "flamegraph":
             collector = FlamegraphCollector(skip_idle=skip_idle)
             filename = filename or f"flamegraph.{pid}.html"
+        case "gecko":
```

Comment:
ℹ️ **Maintainability**

Consider adding a comment to explain the purpose of the 'gecko' case, similar to the other cases. This will improve code readability and maintainability by providing context for future developers.

**Suggested fix:**
```suggestion
        +        # Handle gecko format for Firefox Profiler
        case "gecko":
```